packages/store/src/internal/config-validator.ts

```diff
@@ -1,11 +1,11 @@
-import { Injectable, NgZone, isDevMode, ɵNoopNgZone as NoopNgZone } from '@angular/core';
+import { Injectable, isDevMode } from '@angular/core';
 
 import { isAngularInTestMode } from '../utils/angular';
 import { NgxsConfig } from '../symbols';
 
 @Injectable()
 export class ConfigValidator {
-  constructor(private config: NgxsConfig, private zone: NgZone) {}
+  constructor(private config: NgxsConfig) {}
```

Comment:
```ts
private _config: NgxsConfig
```

Reply:
Not really related to my changes, but I'll do the rename anyway

Reply:
Done